pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/LinuxInfoUtils.java

```diff
@@ -72,6 +73,26 @@ public static boolean isCGroupEnabled() {
         }
     }
 
+    /**
+     *  Get total cpu count.
+     * @return Total cpu count
+     */
+    public static int getTotalCpuCount() throws IOException {
+        int totalCpuCount = 0;
+        String[] ranges = readTrimStringFromFile(Paths.get(CGROUPS_CPU_CPUSET_CPUS)).split(",");
+        for (String range : ranges) {
+            if (!range.contains("-")) {
+                totalCpuCount++;
+            } else {
+                int dashIndex = range.indexOf('-');
+                int left = Integer.parseInt(range.substring(0, dashIndex));
```

Comment:
It may throw `IndexOutOfBoundsException` and `NumberFormatException`, and can't be caught by line#113

Reply:
done.